pkg/scheduler/framework/plugins/apienablement/api_enablement.go

```diff
@@ -37,7 +37,7 @@ func (p *APIEnablement) Filter(ctx context.Context, placement *policyv1alpha1.Pl
 	bindingSpec *workv1alpha2.ResourceBindingSpec, cluster *clusterv1alpha1.Cluster) *framework.Result {
 	if !helper.IsAPIEnabled(cluster.Status.APIEnablements, bindingSpec.Resource.APIVersion, bindingSpec.Resource.Kind) {
 		klog.V(2).Infof("Cluster(%s) not fit as missing API(%s, kind=%s)", cluster.Name, bindingSpec.Resource.APIVersion, bindingSpec.Resource.Kind)
-		return framework.NewResult(framework.Unschedulable, "no such API resource")
+		return framework.NewResult(framework.Unschedulable, "cluster(s) didn't have the API resource")
```

Comment:
why use `(s)` here? I don't understand. It should be always singular.

Reply:
For constructing the diagnosis event, like what `k/k` does. For example:

Warning ScheduleBindingFailed 2s (x10 over 4s) karmada-scheduler 0/3 clusters are available: 1 cluster(s) had untolerated taint {dedicated: special-user}, 2 cluster(s) didn't have the API resource.

Reply:
The number of clusters could be singular or plural.